relayer/src/connection.rs

```diff
@@ -33,16 +32,45 @@ pub use error::ConnectionError;
 /// Maximum value allowed for packet delay on any new connection that the relayer establishes.
 pub const MAX_PACKET_DELAY: Duration = Duration::from_secs(120);
 
-pub fn from_retry_error(e: retry::Error<ConnectionError>, description: String) -> ConnectionError {
-    match e {
-        retry::Error::Operation {
-            error,
-            total_delay,
-            tries,
-        } => {
-            ConnectionError::max_retry(description, tries, total_delay, Box::new(error.0))
+// Includes utility methods and constants to configure the retry behavior
+// for the channel handshake algorithm.
+mod handshake_retry {
+    use crate::connection::ConnectionError;
+    use crate::util::retry::{clamp_total, ConstantGrowth};
+    use core::time::Duration;
+
+    // Approximate number of retries per block
```

Comment:
Let's change the comments on these constants to doc comments.

Reply:
7dd96ac4